_config.yml

```diff
@@ -23,11 +23,16 @@ defaults:
       githubbranch: "master"
       docsbranch: "master"
       versions:
+      - fullversion: "v1.10.0"
+          version: "v1.10"
+          githubbranch: "v1.10.0"
+          docsbranch: "release-1.10"
+          url: https://kubernetes.io/
```

Comment:
1.11 is released, so the above is https://v1-10.docs.kubernetes.io/ - and you need to add 1.11 as well.